mmv1/products/compute/api.yaml

```diff
@@ -6302,6 +6302,13 @@ objects:
         description: |
           An optional description of this resource. Provide this property when
           you create the resource.
+      - !ruby/object:Api::Type::Array
+        name: 'storageLocations'
+        description: |
+          Cloud Storage bucket storage location of the image 
+          (regional or multi-regional). 
+          Reference link: https://cloud.google.com/compute/docs/reference/rest/v1/images
+        item_type: Api::Type::String
```

Comment:
Looks like this value needs to be set to computed as the api is returning defaults for this field.

Reply:
let me check this

Reply:
@ScottSuarez can you give me an example of a property where the computed value is userd?

Reply:
Let me get back to you `default_from_api` should traditionally cover this. Not sure what's going on!

Reply:
ohh I see you just added it in a recent change. Let me rerun tests for you then. Should be covered